app/views/thredded/posts/_content_field.html.erb

```diff
@@ -1,4 +1,6 @@
 <li>
   <%= form.label :content, content_label %>
+  <%= render 'thredded/posts/before_content_field', form: form %>
   <%= form.text_area :content, { rows: 5, required: true }  %>
+  <%= render 'thredded/posts/after_content_field', form: form %>
```

Comment:
I like your reasoning for the name - great point. And thanks for the
reminder on the documentation part. I got lost bouncing around branches
fixing rubocop stuff and forgot to take care of that part :)

On Wednesday, May 11, 2016, Gleb Mazovetskiy <notifications@github.com>
wrote:

> In app/views/thredded/posts/_content_field.html.erb
> <https://github.com/thredded/thredded/pull/293#discussion_r62884576>:
>
> >    <%= form.text_area :content, { rows: 5, required: true }  %>
> > +  <%= render 'thredded/posts/after_content_field', form: form %>
>
> I would say this should be just "after_content" (not
> "after_content_field"), since it's not a given that a field will be
> rendered there, and that this should be documented, but since this is an
> interim solution, LGTM anyway.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/thredded/thredded/pull/293/files/ae6aeabaca82e4d42d68e7d24799e35adab1ecf4#r62884576>
>